### PR TITLE
Paste content to newly created editor

### PR DESCRIPTION
**Pull request by megawebmaster**
Hi!

I've found 2 problems in ngQuill:

1. When user pastes content to newly created editor (i.e. adding a document from Word) underlying ngModel is not updated. This is due to incorrect checking of `newText` value causing initialisation of ngQuill to update editor with empty text (which does not trigger text-changed event).
2. When user selects everything in the editor with Ctrl + A and removes it - ngQuill produces errors. 

My PR resolves both of these problems ;)

As an addition - I've also relaxed AngularJS version checking to avoid "incompatibility may occur" with Angular 1.5 - ngQuill works with it nicely.

### CODE REVIEW

File: src/ng-quill.js [hunk 1]
```diff
@@ -308,7 +308,11 @@
                                     setValue = editor.getHTML();
                                 }
                                 // Set new model value
-                                ngModel.$setViewValue(setValue);
+                                if(editor.getLength() <= 1) {
+                                    ngModel.$setViewValue("");
```
Comment (KillerCodeMonkey) [inline src/ng-quill.js]: would be nice if you could use single quotes at all ;)

Comment (megawebmaster) [inline src/ng-quill.js]: OK, will update in a moment.

**Comment by KillerCodeMonkey** [comment]
thank you so much for contribution :+1:

**Comment by Cordobo** [comment]
+1

**Comment by KillerCodeMonkey** [comment]
released new version 1.4.3

**Comment by megawebmaster** [comment]
Thanks for quick release, this helps a lot!

**Pull request merged**